readme.md

```diff
@@ -27,6 +33,8 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 Your Answer:
 ```text
 
+jQuery allows use to place the ajax calls within the DOM. We call a function on an element and within that function use ajax. The ajax call takes an object as an argument followed by promises determining what will happen on the original element's action.
+
```

Comment:
At a high level we need something( an event) to trigger the ajax, additionally we also need some way to get values out of the form.